packages/SwingSet/test/upgrade/test-upgrade.js

```diff
@@ -253,7 +256,8 @@ test('kernel sends bringOutYourDead for vat upgrade', async t => {
   const expectedDeliveries = [
     messageDeliveryShape(1n, 'getVersion', []),
     deliveryShape(2n, ['bringOutYourDead']),
-    deliveryShape(3n, ['startVat']),
+    // eslint-disable-next-line no-sparse-arrays
+    deliveryShape(3n, ['startVat', ,]),
```

Comment:
I don't understand the context, but I would expect passing a sparse array to fail an assertion somewhere.  Is that what this change is testing?

Reply:
I think this is asserting that the delivery was an array of length 3, whose first element is a `startVat` string, and which doesn't care what the second and third elements are, just as long as they exist. As opposed to the original version, which asserted that it was an array of any length, whose first element is 'startVat'.

In this particular test, I'm not really concerned about checking the length of the array. I wanted to know that a `startVat` happened, and I didn't want to have to fill in the (large, bundle-bearing) details. So I'm not sure this is adding a lot of value.

And the syntax, `['startVat', ,]`, is kinda awful, it looks like a typo or something that `prettier` inserted by mistake. If I really cared about checking the length, I think I'd rather add a third `length` argument to `deliveryShape()`. Or just do the long-form thing I usually do, `d1 = staticVatDeliveries[0]; t.is(d1[0], 'startVat'); t.is(d1.length, 3);`, or something. The helper is nice, but at some point it becomes more hassle than it's worth.

So, I'm kinda -0 on this one.

Reply:
note: `['startVat', undefined, undefined]` would feel better, if it accepts that